lib/rspec/rails/view_controller.rb

```diff
@@ -0,0 +1,43 @@
+module RSpec
+  module Rails
+    # Methods added to ActionView::TestCase::TestController, only in view specs
+    module ViewController
+      def self.included(_klass)
```

Comment:
Unless I'm mistaken, this will permanently include the `ExtraParamsAccessors` methods into ActionView::TestCase::TestController for the rest of the process, including other types of specs. Is my read on this right, and is that the intended behaviour? To me, this looks like it's basically lazy addition of some methods to `ActionView::TestCase::TestController` as opposed to only including some methods in to view specs. If that's the intention, then does the comment need to be changed to better reflect that's whats happening?

PS: If I'm wrong, please let me know, I'm not super familiar with this code ^_^